Error while checking build/O2/fullCI for 49051450b65ccbd1ab88d840ea6f5d49eedf21c2 at 2023-02-24 19:34:
```
## sw/BUILD/O2-latest/log
CMake Error at /sw/slc8_x86-64/CMake/v3.23.1-10/share/cmake-3.23/Modules/FeatureSummary.cmake:464 (message):
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/10834/49051450b65ccbd1ab88d840ea6f5d49eedf21c2/build_O2_fullCI/pretty.html).

--- a/dependencies/O2Dependencies.cmake
+++ b/dependencies/O2Dependencies.cmake
@@ -34,6 +34,14 @@ find_package(Gandiva CONFIG)
 set_package_properties(Arrow PROPERTIES TYPE REQUIRED)
 set_package_properties(Gandiva PROPERTIES TYPE REQUIRED)
 
+if (NOT TARGET Arrow::arrow_shared)
+ add_library(Arrow::arrow_shared ALIAS arrow_shared)
+endif()
+
+if (NOT TARGET Arrow::gandiva_shared)
+  add_library(Gandiva::gandiva_shared ALIAS gandiva_shared)
+endif()
+
 find_package(Vc)
 set_package_properties(Vc PROPERTIES TYPE REQUIRED)
 

--- a/dependencies/O2Dependencies.cmake
+++ b/dependencies/O2Dependencies.cmake
@@ -30,7 +30,7 @@ include(FeatureSummary)
 include(FindThreads)
 
 find_package(Arrow CONFIG)
-if(${Arrow_VERSION} VERSION_LESS 11.0.0)
+if(${Arrow_VERSION} VERSION_LESS 11)
 find_package(Gandiva CONFIG PATHS ${Arrow_DIR} QUIET)
 else()
 find_package(Gandiva CONFIG)

--- a/dependencies/O2Dependencies.cmake
+++ b/dependencies/O2Dependencies.cmake
@@ -30,7 +30,11 @@ include(FeatureSummary)
 include(FindThreads)
 
 find_package(Arrow CONFIG)
+if(${Arrow_VERSION} VERSION_LESS 11.0.0)
+find_package(Gandiva CONFIG PATHS ${Arrow_DIR} QUIET)
+else()
 find_package(Gandiva CONFIG)
+endif()
 set_package_properties(Arrow PROPERTIES TYPE REQUIRED)
 set_package_properties(Gandiva PROPERTIES TYPE REQUIRED)
 

